extras/http2/src/html/macros/helpers.html

```diff
@@ -0,0 +1,11 @@
+{% macro sprite(id, viewBox, classes) %}
+  {% set viewBox = viewBox or "0 0 1 1"%}
+  <span class='sprite -{{id}} {{classes}}'>
+    <svg viewBox="{{viewBox}}"><use xlink:href='images/icons.svg#{{id}}'></use></svg>
+  </span>
+{% endmacro %}
+
+
+{%- macro css(stylesheet) -%}
```

Comment:
If a contributor should update the regular sprite macro (in a future update to blendid) and forget to update the http2 version of the macro, the sprite macro would then be out of sync. Is there a way to have the `-- http2-upgrade` flag inject this css macro into the pre-existing `helpers.html` file?

Reply:
oo thats a good thought. ill see what i can do here